src/app/views/query-response/adaptive-cards/AdaptiveCard.tsx

```diff
@@ -74,10 +75,26 @@ class AdaptiveCard extends Component<IAdaptiveCardProps> {
       } catch (err) {
         return <div style={{ color: 'red' }}>{err.message}</div>;
       }
-    } else {
-        return (<Monaco
-          body={data === null ? `${messages['The Adaptive Card for this response is not available']}` : ''}
-        />);
+    } else if (data === null) {
```

Comment:
This should be the else condition. Since no other condition would exist.

Reply:
but there's the blank condition...
if I ingore the blank condition then the message will be displayed even at startup when no query has been run

Reply:
In that case, what do you think about adding text to tell someone to click run query so that we  display the adaptive card? or possibly fetching the current adaptive card when we load the tab?

Reply:
Yeah I think the message is better

Reply:
So this won't work as the message is displayed first before the adaptive card for the query result is displayed

Reply:
Oh I see what you mean... The adaptive card is dependent on a response body in state. So by checking whether there is a response body, it is possible to display this different view / message.
It could also end up fixing #649

Reply:
Yeah!